Pocket/pocket-ios
# Premium Upgrade View - UI Tests

## Summary
* This PR adds UI tests for Premium Upgrade
> **Note**
There are no failing tests at this moment because setting `failTransactionsEnabled` in `SKTestSession` did not seem to produce the effect of a failed transaction in the store, more investigation to come.

## Implementation Details
* Add accessibility identifiers to `PremiumUpgradeView` and subviews
* Add `PremiumTests` to UI test suite

## Test Steps
* Checkout this branch
* Run snowplow micro locally
* Run the UI tests and make sure they succeed

## PR Checklist:

## Screenshots
NA
